mobile/src/lib/cozy-helper.js

```diff
@@ -35,10 +35,14 @@ export const initClient = (url, onRegister = null, device = 'Device') => {
 }
 
 export const initBar = () => {
+  const root = document.querySelector('[role=application]')
+  const data = root.dataset
```

Comment:
Sorry but you can't do it like that. This method is when app is render by server which replace data.

You can write directly data:

    cozy.bar.init({
      appName: 'Drive',
      appEditor: 'Cozy',
      iconPath: require('../../../vendor/assets/app-icon.svg'),
      lang: getLang()

getLang is on this PR https://github.com/cozy/cozy-drive/pull/231